### PR TITLE
Unit test for notifications

### PR DESCRIPTION
**Pull request by Devaansh-Kumar**
**What type of PR is this?**

/kind test

**What this PR does / why we need it**:
This PR adds unit tests for notifications package.

**Which issue(s) this PR fixes**:

This pull request addresses part of the issue described in #131 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/cc @LiorLieberman @mlavacca

### CODE REVIEW

**Review by LiorLieberman — COMMENTED** [review]
/ok-to-test

File: pkg/i2gw/notifications/notifications_test.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+			}
+			result := na.CreateNotificationTables()
+			for provider, table := range result {
+				assert.Equal(t, tc.wantedTables[provider], table)
+			}
+		})
+	}
+}
+
+func Test_convertObjectsToStr(t *testing.T) {
```
Comment (mlavacca) [inline pkg/i2gw/notifications/notifications_test.go:156]:
```suggestion
func TestConvertObjectsToStr(t *testing.T) {
```

**Review by mlavacca — APPROVED** [review]
Other than a minor naming thing, lgtm!

**Review by LiorLieberman — APPROVED** [review]
Great stuff, thanks @Devaansh-Kumar !
/lgtm

**Pull request merged**